CreeDictionary/phrase_translate/crk_translate_test.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+
+from phrase_translate.translate import inflect_english_phrase
+
+WAPAMEW_DEFINITION = "s/he sees s.o."
+
+
+@pytest.mark.parametrize(
+    ("analysis", "definition", "english_phrase"),
+    (
+        ("masinahikan+N+Dim+Px1Sg+Loc", "book", "in my little book"),
+        ("atâhk+N+A+Der/Dim+N+A+Obv", "star", "little star over there"),
+        (
+            "nêhiyawasinahikan+N+I+Der/Dim+N+I+Px1Sg+Loc",
+            "Cree book",
+            "in my little Cree book",
+        ),
+        (
+            "PV/e+wîtatoskêmêw+V+TA+Cnj+1Sg+2SgO",
+            "s/he works together with s.o.",
+            "I work together with you",
+        ),
+        (
+            "PV/e+wîtapimêw+V+TA+Cnj+1Sg+2SgO",
+            "s/he sits with s.o., s/he stays with s.o., s/he is present with s.o.",
+            "I sit with you, I stay with you, I am present with you",
+        ),
+        ("wâpamêw+V+TA+Fut+Cond+2Sg+3SgO", WAPAMEW_DEFINITION, "when you see him/her"),
+        (
+            "wâpamêw+V+TA+Imp+Imm+12Pl+3SgO",
+            WAPAMEW_DEFINITION,
+            "let you and us see him/her now",
+        ),
+        ("wâpamêw+V+TA+Imp+Del+2Sg+3PlO", WAPAMEW_DEFINITION, "let you see them later"),
+        (
+            "PV/ki+wâpamêw+V+TA+Ind+2Pl+1PlO",
+            WAPAMEW_DEFINITION,
+            "you all saw us",
+        ),
+        (
+            "PV/wi+wâpamêw+V+TA+Ind+4Sg/Pl+3PlO",
+            WAPAMEW_DEFINITION,
+            "s/he/they want to see them",
+        ),
+        (
+            "PV/e+PV/wi+wâpamêw+V+TA+Cnj+1Sg+3PlO",
+            WAPAMEW_DEFINITION,
+            "I want to see them",
+        ),
+        (
+            "PV/ka+wâpamêw+V+TA+Ind+4Sg/Pl+3SgO",
+            WAPAMEW_DEFINITION,
+            "s/he/they are going to see him/her",
+        ),
+        (
+            "PV/ka+wâpamêw+V+TA+Cnj+1Sg+2PlO",
+            WAPAMEW_DEFINITION,
+            "for me to see you all",
+        ),
+        (
+            "PV/ta+wâpamêw+V+TA+Cnj+3Sg+1PlO",
+            WAPAMEW_DEFINITION,
+            "for him/her to see us",
+        ),
+        (
+            "atamihêw+V+TA+Ind+4Sg/Pl+2PlO",
+            "s/he makes s.o. smile",
+            "s/he/they make you all smile",
+        ),
```

Comment:
What happens with lexicalized forms like [kitatamihin](https://itwewina.altlab.app/search?q=kitatamihin)? Arok lists it as an [IPC with its own definition](https://itwewina.altlab.app/word/kitatamihin/?pos=IPC), but it _is_ [an inflected form of atamihêw](https://itwewina.altlab.app/word/atamih%C3%AAw/#paradigm:~:text=kitatamihin).

Reply:
For such a form, you should get both 1) the phrase translation, understood as an inflected form of atamihêw, and 2) the manually provided translation for that particular inflected word form.

`echo 'kitatamihin' | inc/eng2crk-phrase-transcriptor.sh /Users/arppe/altlab/crk/dicts/Wolvengrey.tsv transcriptor-eng-phrase2crk-features.fomabin crk-anl-desc-dict.fomabin crk-gen-norm-dict.fomabin transcriptor-cw-eng-noun-entry2inflected-phrase-w-flags.fomabin transcriptor-cw-eng-verb-entry2inflected-phrase-w-flags.fomabin WA_fst_cg.frequency_list.txt
`
```
kitatamihin (<- kitatamihin)
=> atamihêw [VTA-1]
=> _+V+TA+Ind+2Sg+1SgO
-----
     ≈  you make me smile; you please me, you make me glad; you make me grateful, you make me indebted; you treat me well [VTA]

```

Reply:
Yup, `translate.py` outputs

    wordform: kitatamihin ['+V', '+TA', '+Ind', '+2Sg', '+1SgO']
    lemma: atamihêw+V+TA+Ind+3Sg+4Sg/PlO
        definition: s/he makes s.o. smile; s/he pleases s.o., s/he makes s.o. glad; s/he makes s.o. grateful, indebted; s/he treats s.o. well →
        you make me smile; you please me, you make me glad; you make me grateful, indebted; you treat me well
        definition: he/she makes him/her/them smile; he/she pleases him/her/them, he/she makes him/her/them glad; he/she makes him/her/them grateful, indebted; he/she treats him/her/them well →
        you make him/her/them smile; you please him/her/them, you make him/her/them glad; you make him/her/them grateful, indebted; you treat him/her/them well
    wordform: kitatamihin ['+Ipc', '+Interj']
    lemma: kitatamihin+Ipc+Interj
        definition: thank you, you make me smile, you please me →
        (not supported)

The +Ipc is not supported as there are only transcription rules for nouns and verbs. At the moment, the “you make me smile …” auto-definition would not show up on the website. I would need to investigate why. We know that with more work we could translate more things. Where that falls on the priority list I’m not quite sure, but I would leave any improvements there to a future pull request.

Reply:
Since _kitatamihin_ has two analyses, one of which is an inflected verb form and another a lexicalization treating the form as in Interjection (a subset of Indeclinable particles), then the verbal analysis should produce the generated translated English phrase, whereas the other analysis shouldn't, as it is "indeclinable" and the particular inflected verb form does have the specific translation provided manually by Arok.